package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "huron-example-setup",
+  "version": "1.0.0",
+  "description": "An example site using the Huron prototyping tool.",
+  "main": "index.js",
+  "scripts": {
+    "//" : "The Huron script runs the devserver so you can build prototypes on the fly",
```

Comment:
Interesting, alright! Might be clearer to document recommended scripts in the `README.md`